replicas/replicas_calculator.go

```diff
@@ -38,10 +38,16 @@ func (rc *ReplicaCalculator) CalculateReplicas(currentReplicas int32, metricTarg
 		return rc.calculateValue(currentReplicas, metricTarget, metricValues)
 	case v1alpha1.AverageValueMetricType:
 		return rc.calculateAverageValue(currentReplicas, metricTarget, metricValues)
+		// case v1alpha1.UtilizationMetricType:
```

Comment:
any reasons for commenting them out?

Reply:
They got mixed with a different patch that implements them. I'll clean it up